docs: Update CONTRIBUTING.md (#50)
charmed-hpc/docs: CONTRIBUTING.md

the right place then! __Here is how you can get involved.__

Please take a moment to review this document so that the contribution
process will be easy and effective for everyone. Following these guidelines helps you communicate that you respect the maintainers
managing Charmed HPC's documentation. In return, they will reciprocate that respect

> while addressing your issue or assessing your submitted changes and/or features. 

Reviewer: [nit]: extraneous whitespace here :sweat_smile: 

```suggestion
while addressing your issue or assessing your submitted changes and/or features.
```

## Issues and Labels

The Charmed HPC documentation issue tracker uses a variety of labels to help organize
and identify issues. Here are a few of these labels, and how the maintainers of
Charmed HPC's documentation use them:

* `docs` - Issues for general revision and enhancement of Charmed HPC's documentation.
Can also be used for pull requests.

* `help wanted` - Issues where we need help from the HPC community to solve.

> * `good first issue` - Issues that the Charmed HPC documentation maintainers have determine are a suitable level for first time documentation contributors. 

Reviewer: I think we want `determined` here:

```suggestion
* `good first issue` - Issues that the Charmed HPC documentation maintainers have determined to be suitable for first time contributors. 
```

I removed the "documentation" qualifier for contributors because we want everyone to contribute to documentation, not just the "technical authorship" inclined!

documentation. Good bug reports make Charmed HPC's documentation better, so
thank you for taking the time to report issues!

> **_NOTE:_** For bug reports pertaining to code functionality, please post the bug report in the corresponding code repository rather than here. Bug reports here should focus on bugs within the documentation, such as a how-to command being displayed incorrectly, or the documentation setup itself not functioning as expected. 

> 

Author: Very true. The original warning was more of a note but I tweaked it to be stronger and I agree it shouldn't be a note anymore.

    # Ensure style guide compliance
    $ make vale
    ```
> **_NOTE:_** The current MAKEFILE setup assumes that you are using an Ubuntu OS. If not, or if running the tests locally is not ideal, you may run them within GitHub. To do so, make sure any local changes have been pushed to your personal fork+branch and are visible from the web interface, then, from the web interface for GitHub:
>  1. Go to the `Actions` tab near the top of the screen
>  2. Select `Automatic docs checks` on the far left
>  3. Select the `Run workflow` button on the right
>  4. Select the relevant branch from the drop-down menu
>  5. Select `Run workflow` within the drop-down
>

> > For more information on setting up the tests locally, see [Automatic checks](https://canonical-starter-pack.readthedocs-hosted.com/latest/content/automatic_checks/) within the Canonical Starter Pack documentation.

Author: Agreed, if there was a different style of admonition I thought would work correctly, I'd use one of those instead. We could just remove the 'Note:' part, but leave the blocking - shows that it's a slight context switch from the rest of the text it's in but doesn't denote that it's optional the way saying 'Note' implies.
